src/variable.cpp

```diff
@@ -24,60 +24,6 @@ template <class T1, class T2> bool equal(const T1 &view1, const T2 &view2) {
 template <class T> class DataModel;
 template <class T> class VariableConceptT;
 template <class T> struct RebinHelper {
-  static void rebin(const Dim dim, const gsl::span<const T> &oldModel,
```

Comment:
What was the reason why this `rebin` is now broken?

Reply:
It always has been, unless rebinning the inner dimension. I thought it was better to just disable it for now.